internal/command/apply.go

```diff
@@ -379,6 +379,9 @@ Options:
   -parallelism=n         Limit the number of parallel resource operations.
                          Defaults to 10.
 
+  -replace=resource     Terraform will plan to replace this resource instance
+                        instead of doing an update or no-op action. 
+
```

Comment:
```suggestion
  -replace=resource      Terraform will plan to replace this resource instance
                         instead of doing an update or no-op action. 

```

I noticed a slight offset when printing the help text. Adding a space should fix it

![CleanShot 2025-04-03 at 11 33 31@2x](https://github.com/user-attachments/assets/584fc95d-01df-4a46-a9e6-94ee93fa5937)